apps/starwars/priv/static/css/app.css

```diff
@@ -0,0 +1,1275 @@
+/*
```

Comment:
This file should not be under version control because it's generated from webpack